distribution/ddf-common/src/main/resources/bin/start

```diff
@@ -23,17 +23,32 @@
 # ##############################################################################
```

Comment:
❓ Is this comment block still valid? Pasted for convenience:
```
# ##############################################################################
# The only change in this script from Karaf's original script is to call ddf
# instead of karaf at the end of the file to gain restart support for critical
# and out of memory errors
# ##############################################################################
```

Reply:
❗️ It should be. Whatever changes was done, this logic should be re-inserted.